packages/salesforcedx-vscode-core/package.json

```diff
@@ -43,7 +43,7 @@
     "@types/chai": "^4.0.0",
     "@types/glob": "^5.0.30",
     "@types/mocha": "^5",
-    "@types/node": "8.9.3",
+    "@types/node": "^13.11.0",
```

Comment:
Do we want to use a version 12 flavor of these types since VS Code stable and insiders is on 12.8.1 and 12.13.0 respectively?

Reply:
> VS Code stable and insiders is on 12.8.1 and 12.13.0 respectively?
Where could I find this?

`^13.11.0` comes from the version of `@types/node` in a new projected scaffolded by yo https://code.visualstudio.com/api/get-started/your-first-extension

Reply:
Ah interesting. If you go to the "About" section in the menu bar this shows up:

![Screen Shot 2020-06-23 at 3 52 24 PM](https://user-images.githubusercontent.com/10779873/85474440-8a9d9c80-b569-11ea-909e-1902bfa3b3c5.png)

And on the master branch of the vscode repo they're using ^12.11.7

Reply:
Yeah that's interesting. I think we should use 12 for now. Not sure about the specific version though.
Our node dependency is [on 12.4.0](https://github.com/forcedotcom/salesforcedx-vscode/pull/1884/files). We could use that or update both node dependency and `@types/node` to `12.8.1`.

@lcampos thoughts?

Reply:
@brpowell @xyc the node version decision comes from the Electron version being shipped by the min VSCode version (engine in package.json) that we decide to support. VSCode 1.43 is the one that started shipping with NodeJS version 12.8.1 (https://code.visualstudio.com/updates/v1_43#_electron-70-update). We should only merge these changes if we can merge a change to the extension's engine version to be 1.43 and above in the same week/PR.

If we can't do that then let's stick to @types/node being 12.0.12 (I don't see a version equal to 12.4.0).

Reply:
Makes sense. Let's address engine update separately. Changed `@types/node` version to `12.0.12`.